AllReadyApp/Web-App/AllReady/Features/Resource/ResourcesByCategoryQueryHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using AllReady.Models;
+using MediatR;
+
+namespace AllReady.Features.Resource
+{
+    public class ResourcesByCategoryQueryHandler : IRequestHandler<ResourcesByCategoryQuery, List<Models.Resource>>
+    {
+        private readonly IAllReadyDataAccess dataAccess;
+
+        public ResourcesByCategoryQueryHandler(IAllReadyDataAccess dataAccess)
+        {
+            this.dataAccess = dataAccess;
+        }
+
+        public List<Models.Resource> Handle(ResourcesByCategoryQuery message)
+        {
+            return dataAccess.GetResourcesByCategory(message.Category).Distinct().ToList();
```

Comment:
@mheggeseth, the original query to dataAccess.GetReorcesByCategory was in ResourceApiController and looked like this:
`var resources = (from c in _allReadyDataAccess.GetResourcesByCategory(category) select c).Distinct();`

although the query does't really make sense with `Distinct`, I don't know enough about the entities and their relationships yet to make that call on not including it.

So when I refactored the query out of ResourceApiController to the handler, I kept it there.

Do you want me to remove it?